### PR TITLE
Update charter given feedback in #2, #3, #6, #8, and #10.

### PR DESCRIPTION
**Pull request by hober**
From #2, especially @travisleithead's comment:

- [x] Add text to the charter describing the pre-deliverable proposal phase.
- [x] Tweak example destinations (drop WebAppSec?)
- [x] Rename "deliverable" to something less serious sounding

From #3:

- [x] use the word incubation
- [x] Refrain from calling things standards or specifications
- [x] Define the lifecycles of proposals and deliverables
    - [x] describe exit conditions for an incubation
    - [x] for how / whether a proposal should be abandoned/dropped from the group
    - [x] or how a proposal should complete incubation

### CODE REVIEW

**Comment by hober** [comment]
I've attempted to capture a whole bunch of this feedback in 0d58d27:

* use the word "incubate" more places
* renamed "deliverable" to "work item"
* added section defining (pre-work item) proposals and their lifecycle
* defined the lifecycle of a work item
* dropped use of the words "standard" or "specification" when talking about work items

File: charter.html [hunk 1]
```diff
@@ -236,28 +259,29 @@ <h2 id=coordination>Coordination</h2>
...
 
 <dl>
 <dt><a href=https://www.w3.org/Privacy/IG/>Privacy Interest Group (PING)</a>
 <dd>This group will coordinate with PING and will take into
-consideration outputs of PING when evaluating proposals.
+consideration outputs of PING when
+evaluating <a href=#proposals>Proposals</a> and Work Items.
 <dt><a href=https://www.w3.org/2011/webappsec/>Web Application Security
 Working Group (WebAppSec)</a>
 <dd>WebAppSec is expected to be the main destination for transitioning
```
Comment (TanviHacks) [inline charter.html]: change to "WebAppSec is expected to be a destination for transitioning" instead of using the word "main".

File: charter.html [hunk 2]
```diff
@@ -236,28 +259,29 @@ <h2 id=coordination>Coordination</h2>
...
+this group’s independent Work Items to the standards track.
 <dt><a href=https://www.w3.org/community/wicg/>Web Platform Incubator
 Community Group (WICG)</a>
-<dd>WICG is expected to be a major source of Deliverables for this
+<dd>WICG is expected to be a major source of Work Items for this
 group.
-<p class=note>Only privacy-related WICG work items which have the
-support of at least two <a href=#implementer>implementers</a> are
-eligible for this group to take up as Deliverables.
+<p class=note>Only privacy-related WICG proposals which have the support
```
Comment (TanviHacks) [inline charter.html]: I wonder if we should remove this note given we have the two implementer support called out in other places.

Comment (hober) [reply charter.html]: It's a note because the normative text that sets this requirement is elsewhere. I think it's mildly helpful as a reminder, but I think it'd be okay to remove too.

File: charter.html (hunk 1 above)
Comment (hober) [reply charter.html]: I've updated this wording in d5951a1. LMK if the new text works for you.

**Comment by hober** [comment]
Progress update:

in 66c8fec:
* added the TAG to the list of groups to coordinate with.
* Require TAG review of work items at some point during their lifecycle.
* Add examples of PING and TAG output that should be considered when working on proposals and work items.

in 43b6aeb and 5d4b6fe:
* Clarify lifecycle of proposals and work items

Still to do:
* give guidance on figuring out a likely destination standards body
* describe / define the Chairs' role in focusing the group on particular proposals or work items

File: charter.html [hunk 3]
```diff
@@ -206,7 +206,7 @@ <h2 id=proposals>Proposals</h2>
 </ul>
 
 <p>Proposals may be withdrawn by their originators, or may be closed by
-the Chairs if they fail to gain
+the Chairs—if, for example, they fail to gain
```
Comment (TanviHacks) [inline charter.html]: add space before "if".

File: charter.html (hunk 1 above)
Comment (TanviHacks) [reply charter.html]: Looks good!

File: charter.html (hunk 3 above)
Comment (hober) [reply charter.html]: Is the em dash not sufficient?

Comment (TanviHacks) [reply charter.html]: I suppose.  Perhaps a stylistic change.

File: charter.html (hunk 2 above)
Comment (TanviHacks) [reply charter.html]: @erik-anderson?

File: charter.html (hunk 3 above)
Comment (hober) [reply charter.html]: Switched it to parens in 4ea0616.

File: charter.html [hunk 4]
```diff
@@ -207,24 +238,32 @@ <h3 id=specifications>Specifications</h3>
...
+
+<p class=note>Typically, Work Items begin life as a Proposal before
+being adopted.
+
+<p>When a Work Item's Editors deem the Work Item ready for migration to
+a standards body, they will notify the Chairs. The Editors and Chairs
+will then work together to identify the best destination, and will work
+with the destination standards body to successfully migrate the work.
+
+<p>The Chairs may remove Work Items. The Chairs must announce the
```
Comment (jyasskin) [inline charter.html]: What forum should folks watch to get these announcements?

File: charter.html [hunk 5]
```diff
@@ -417,23 +464,24 @@ <h3 id=decisions>Decision Policy</h3>
 <p>Implementation disagreement should not result in
 implementation-defined behavior or optional features.
 
-<p>Specifications should not make references to or rely on specific
```
Comment (jyasskin) [inline charter.html]: I think "specifications" was fine here. CGs are [explicitly allowed to produce specifications](https://www.w3.org/community/about/agreements/).

File: charter.html [hunk 6]
```diff
@@ -178,24 +179,54 @@ <h3 id=chair-selection>Chair Selection</h3>
...
-<p>The <a href=#chairs>Chairs</a> may add Deliverables, but must not add
-Deliverables which lack the support of at least
-two <a href=#implementer>implementers</a>.
+<p>Proposals may be withdrawn by their originators, or may be closed by
+the Chairs (if, for example, they fail to gain
+sufficient <a href=#implementer>implementer</a> support to be adopted as
+a Work Item).
 
-<p>The Chairs may remove Deliverables.
+<h2 id=work-items>Work Items</h2>
```
Comment (jyasskin) [inline charter.html]: Are these "Draft Community Group Reports" in the terminology of https://www.w3.org/community/about/agreements/#cgroups?

File: charter.html [hunk 7]
```diff
@@ -178,24 +179,54 @@ <h3 id=chair-selection>Chair Selection</h3>
 <a href=https://www.w3.org/People/functions/w3m#dom>Dominique
 Hazaël-Massieux</a>.
 
-<h2 id=deliverables>Deliverables</h2>
+<h2 id=proposals>Proposals</h2>
```
Comment (jyasskin) [inline charter.html]: Is there a concrete difference between how the CG will interact with a Proposal vs a Work Item? It looks like a sufficiently complex proposal can get a repository before multiple engines have expressed interest, and the CG discusses them similarly.

Would it shorten or simplify the charter to say that everything's a Proposal, and the chairs will keep a list of proposals for which 2 engines have expressed interest?

File: charter.html [hunk 8]
```diff
@@ -178,24 +203,55 @@ <h3 id=chair-selection>Chair Selection</h3>
...
+<li>because the Proposal is sufficiently complex that having a dedicated
+issue tracker for it is warranted
+<li>because multiple <a href=#implementer>implementers</a> have
+expressed interest in the Proposal
+</ul>
+
+<p>Proposals may be withdrawn by their originators, or may be closed by
+the Chairs (if, for example, the Chairs deem the Proposal to
+be <a href=out-of-scope>out of scope</a> or the Proposal fails to gain
+sufficient <a href=#implementer>implementer</a> support to be adopted as
```
Comment (jyasskin) [inline charter.html]: I hate to suggest adding text, but when the chairs decide to close a proposal that has a dedicated repository, it would be good to have assurance that y'all will make an effort to transfer the repository to another org or user, instead of just deleting the data in the repository.

File: charter.html (hunk 7 above)
Comment (travisleithead) [reply charter.html]: The major difference seems to be around the allocation of [potentially scarce] Editors, which are appointed by the chairs (do these editors have to be members of the CG?) to advance the work item. The Editors then takeover much of the responsibility of "directing the work" until they judge its ready to advance beyond the CG. Did I get that right?

What does this mean for proposals that have a highly-engaged and willing "owner"/"creator"? If the chairs do not want to see the proposal advance (or the proposal doesn't meet the implementer criteria), I'm not sure they could do much to stop a pro-active proposal owner (not an officially minted editor) from carrying on. It seems the purpose of promotion to Work Item then is to lend a helping hand to good proposals with less-active owners and get them moving even faster in the process. Might be worth calling out the rationale a bit more.

Comment (hober) [reply charter.html]: Hmm. The intent is that the output of the proposal phase is an explainer, and the output of the work item phase is a revised explainer and a draft spec. This is spelled out in the proposals repo https://github.com/privacycg/proposals but not in the charter. I'll try to figure out how to incorporate that here. Thanks!

File: charter.html (hunk 6 above)
Comment (hober) [reply charter.html]: yes, CG-DRAFT in Bikeshed-ese.

Comment (hober) [reply charter.html]: "Draft Community Group Reports" is much wordier than "Work Item", though. I mean, "Work Item" is already too wordy.

File: charter.html (hunk 5 above)
Comment (hober) [reply charter.html]: Personally, I agree. But given the strongly worded feedback in #3, I'd rather avoid the word.

File: charter.html (hunk 4 above)
Comment (hober) [reply charter.html]: Yeah, we should probably spell that out. Good catch. I've added text for this in 7e59061.

File: charter.html [hunk 9]
```diff
@@ -178,24 +203,55 @@ <h3 id=chair-selection>Chair Selection</h3>
...
+teleconferences or face-to-face meetings.
 
-<p>The Chairs may remove Deliverables.
+<p>The <a href=#chairs>Chairs</a> may create a dedicated repository for
+a Proposal. They may do this for any reason, such as but not limited to:
 
-<p>This document will be updated to reflect the current set of
-Deliverables of the Community Group.
+<ul>
+<li>because the Proposal has become sufficiently complex that writing an
```
Comment (travisleithead) [inline charter.html]: I imagine that a proposal author might just want to proactively create an explainer, and would naturally drop it into the proposals repo--so no need to get their own repo just to do that. Having a formal explainer, even in the proposal repo, allows the proposer an opportunity to refine and adapt the proposal if it is (for example) rejected by the chairs, or not enticing to enough implementers.

File: charter.html (hunk 8 above)
Comment (hober) [reply charter.html]: > I hate to suggest adding text,

hahahaha, for real, this thing is already quite long.

> but when the chairs decide to close a proposal that has a dedicated repository, it would be good to have assurance that y'all will make an effort to transfer the repository to another org or user, instead of just deleting the data in the repository.

Yeah, this is a good point. I've tried to address this in a319a16.

File: charter.html (hunk 6 above)
Comment (hober) [reply charter.html]: (and Proposals are not Draft Community Group Reports, they are just ideas someone had that may or may not go anywhere.)

File: charter.html (hunk 5 above)
Comment (travisleithead) [reply charter.html]: Well, community group reports, really.

File: charter.html (hunk 9 above)
Comment (hober) [reply charter.html]: They could even simply use the explainer as the text of the issue. I figure that's how a bunch of explainers will happen—the issue gets edited over time to reflect the discussion, and it ends up more and more explainer-y.

Comment (travisleithead) [reply charter.html]: Yep, that's what I figured you had in mind. I was just thinking that if a proposer wants to allow PRs and better change-tracking, they would want to make a separate code file--they're not limited to 1) tracking proposals as issues or 2) waiting to request a repo from the chairs.

File: charter.html (hunk 2 above)
Comment (erik-anderson) [reply charter.html]: I have no preference.

File: charter.html (hunk 7 above)
Comment (hober) [reply charter.html]: I'm going to raise this as a separate issue.

File: charter.html (hunk 9 above)
Comment (hober) [reply charter.html]: Sure, we could possibly do that too, I suppose. Do you think we need to call that out in the charter?

File: charter.html [hunk 10]
```diff
@@ -93,19 +93,20 @@ <h1>Privacy Community Group Charter</h1>
 <h2 id=mission>Mission</h2>
 
 <p>The mission of the <a href=https://privacycg.github.io/>Privacy
-Community Group</a> is to incubate privacy-focused web standards and APIs
+Community Group</a>, motivated by the
+<a href=https://w3ctag.github.io/ethical-web-principles/>W3C TAG Ethical
```
[the remark this answers is not in the document]
Comment (hober) [reply charter.html]: Fixed in 2790960.

**Comment by hober** [comment]
Okay, I think this is a good batch of updates. I'd like to land it, once @erik-anderson approves. There is a bit of followup work; see #11.

File: charter.html [hunk 11]
```diff
@@ -207,24 +275,37 @@ <h3 id=specifications>Specifications</h3>
...
+<li>because the Work Item has been migrated elsewhere
+<li>because the Work Item no longer has the support of
+multiple <a href=#implementer>implementers</a> and is unlikely to regain
+it
+<li>because the Work Item no longer has an <a href=#editors>Editor</a>
+and a replacement cannot be found
+</ul>
+
+<p>The Chairs should take steps to ensure the repositories of removed
+Work Items are not lost, perhaps by transfering the repository to a
```
Comment (jyasskin) [inline charter.html]:
```suggestion
Work Items are not lost, perhaps by transferring the repository to a
```

File: charter.html [hunk 12]
```diff
@@ -285,8 +371,8 @@ <h2 id=process>Process</h2>
 <a href=https://www.w3.org/Consortium/cepc/>W3C Code of Ethics and
 Professional Conduct</a>.
 
-<p>Contributions to Deliverables can only be made by Community Group
-Participants who have agreed to the
+<p>Contributions to <a href=#work-items>Work Items</a> can only be made
```
Comment (jyasskin) [inline charter.html]: This is probably also true for Proposals. You'll want a clear IP trail for everything in the group.

Comment (hober) [reply charter.html]: good catch; fixed in b641830.

File: charter.html (hunk 11 above)
Comment (hober) [reply charter.html]: thanks, fixed
